low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -0,0 +1,46 @@
+#!/bin/python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com
+
+"""
+ ***************************************************************************
+  Description: Server class to provide fru health and specific information
+ ***************************************************************************
+"""
+
+from resource_map import ResourceMap
+
+
+class ServerMap(ResourceMap):
+    """Provides health information of FRUs in a node"""
```

Comment:
This description can be updated too, to 
""" Provides Server resource map & related info like health, manifest, etc """

Reply:
```

"""
 ***************************************************************************
  Description: ServerMap class provides resource map and related information
               like health, manifest, etc,.
 ***************************************************************************
"""
```
Added in module description.
Class doc string,
```
class ServerMap(ResourceMap):
"""Provides server resource related information."""
```